common/source/docs/common-cuav-nora-overview.rst

```diff
@@ -70,10 +70,10 @@ Default UART Order
 ==================
 
 - SERIAL0 = console = USB
-- SERIAL1 = Telemetry1 = USART1
-- SERIAL2 = Telemetry2 = USART6
-- SERIAL3 = GPS1 = USART1
-- SERIAL4 = GPS2 = UART4
+- SERIAL1 = TELEM1 = USART2
+- SERIAL2 = TELEM2 = USART6
+- SERIAL3 = GPS = USART1
+- SERIAL4 = UART4 = UART4
```

Comment:
sorry,this is not correct...default protocol for SERIAL4 is GPS2..I can agree on the changes to SERIAL1 and 2 but not 3 and 4